frontends/election-manager/src/screens/unconfigured_screen.tsx

```diff
@@ -79,16 +79,12 @@ export function UnconfiguredScreen(): JSX.Element {
     history.push(routerPaths.electionDefinition);
   }
 
-  const saveElectionAndShowSuccess = useCallback(
-    (electionJson: string) => {
+  const saveElectionSafely = useCallback(
```

Comment:
Not a blocker but… why "safely"?

Reply:
I can rename it if that's not obvious, I meant because it validates the data model.

Reply:
Ah. Just a little weird since it then calls `unsafeUnwrap`.
![is it safe](https://user-images.githubusercontent.com/1938/191306658-44bcffbe-0774-42b4-bb4b-e88fe2519887.jpeg)

Reply:
oooh, I think I misread that! So that's ... not validating the schema? Something else is?

Reply:
See, it is confusing! It is validating the data and returning a `Result`, which we then `unsafeUnwrap` which will throw if the validation failed. The "safe" part refers to the fact that the function doesn't `throw`.

Reply:
ahah! OK so I'm doing it right but maybe using the word `safe` is just confusing here and I'll use something clearer, like `validateAndSaveElection`